src/subworkflows/convergence.js

```diff
@@ -1,8 +1,33 @@
-import { UNIT_TYPES } from "../enums";
+import { UNIT_TAGS, UNIT_TYPES } from "../enums";
 import { createConvergenceParameter } from "./convergence/factory";
 
 export const ConvergenceMixin = (superclass) =>
     class extends superclass {
+        get convergenceParam() {
+            return this.findUnitWithTag("hasConvergenceParam")?.operand || undefined;
+        }
+
+        get convergenceResult() {
+            return this.findUnitWithTag("hasConvergenceResult")?.operand || undefined;
+        }
+
+        // TODO: investigate how scope changes between subworkflows to allow for multiple convergences per job, in different subworkflows
+        convergenceSeries(scopeTrack) {
+            if (!this.hasConvergence || !scopeTrack?.length) return [];
+            let lastResult;
+            const series = scopeTrack
+                .map((scopeItem) => ({
+                    x: scopeItem.scope.global[this.convergenceParam],
```

Comment:
Probably would make sense to have `?` after scope, global to make this safe

Reply:
added optional chaining `scopeItem.scope?.global[]`